provisioning/staging/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM maelstromeous/applications:ps2alerts-websocket-base
+
+USER root
+
+COPY provisioning/staging/files/entrypoint.sh /etc/service/entrypoint.sh
+
+USER node
+
+COPY --chown=node:node ./app /app
+
+RUN cd /app && npm install && npm run build
```

Comment:
Instead of npm install use npm ci. After you have run build use npm prune —production.

Reply:
https://github.com/ps2alerts/websocket/pull/74/commits/a3bf904ddaea0c0949c8b2cb78d77da08437f993